### DISCUSSION TITLE
DoT is not working on Android 9+

### DISCUSSION THREAD

**Discussion by isac322** [0xERR0R/blocky#342] [Q&A] [answered]
First of all, thanks for your work. It's very easy to setup HA dns blocker.
I have two android 11 devices so I deployed e882fa0 (contains #306) to my k8s cluster. When I run `dog https://pagead2.googlesyndication.com -S @dnbs.bhyoo.com` ([dog: dns client](https://github.com/ogham/dog)) on linux machines, there are no error and blocks perfectly. but when I apply it to Android devices ([DoT example for Android 9+](https://blog.cloudflare.com/enable-private-dns-with-1-1-1-1-on-android-9-pie/)) it failed without any debug log on blocky pod. Please help!

**Comment by 0xERR0R** [comment]
Do you use let's encrypt certificate?

**Comment by isac322** [comment]
> Do you use let's encrypt certificate?

Yeap I created it using cert-manager.
FYI. I didn't setup DoH and use only DoT.

**Comment by 0xERR0R** [comment, marked as answer]
Ok, please try to generate new certificate with "ISRG Root X1" chain (there should be a parameter).

**Comment by isac322** [comment]
@0xERR0R It worked thanks!!

**Comment by 0xERR0R** [comment]
I had the same issue with my Android. I think, le changed the root certificate and android can't validate it.
